frontend/src/components/customers/CustomerHome.js

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { useHistory } from "react-router-dom";
+import axios from "axios";
+import { Container, Col, Row, Spinner } from "react-bootstrap";
+import "../../style/customer-home.scss";
+import ActiveReservation from "./ActiveReservation";
+
+function CustomerHome(props) {
+  const [activeReservations, setActiveReservations] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [empty, setEmpty] = useState(false);
+  const [serviceNames, setServiceNames] = useState(null);
+  const [stylistNames, setStylistNames] = useState(null);
+  const history = useHistory();
+
+  // TODO: Proper error handling
+
+  // Get active reservations, service names from id list, and stylist name from id
+  useEffect(() => {
+    setLoading(true);
```

Comment:
For readability, I would suggest extracting some of this into functions to call.

Reply:
Or at the very least divide the code visually with a comment explaining what happens underneath.

Reply:
Done